modules/logger/logger.py

```diff
@@ -11,7 +11,8 @@
 
 # Used in type annotation of logger parameters
 # pylint: disable-next=unused-import
-import types
```

Comment:
My bad, I just realized this is used elsewhere, can you bring it back? (Also add a space between this and import numpy)